### ISSUE TITLE
are there any plans to add order independent transparency?

### ISSUE THREAD

**Issue by RallyTronics** [pniekamp/datum#5]

**Comment by pniekamp** [comment]
I was thinking about adding a weighted blend pass for particles and transparent objects.

Is that the sort of thing you are thinking about ?

**Comment by RallyTronics** [comment]
Yeah something like that, I found this presentation quite interesting: http://on-demand.gputechconf.com/gtc/2014/presentations/S4385-order-independent-transparency-opengl.pdf
